src/common/UserAvatarWithPresence.js

```diff
@@ -36,7 +36,8 @@ type Props = $ReadOnly<{|
  */
 export default class UserAvatarWithPresence extends PureComponent<Props> {
   static defaultProps = {
-    avatarUrl: '',
+    // TODO: An empty-string `email` is probably up to no good. Remove
+    // this default.
     email: '',
```

Comment:
Also make a note about another default prop that would be good to
    remove [1], but that is still operating in one case.

Dang.

It looks like that's `LightboxHeader`... and I think after the changes in this PR (so that the email isn't being used here to find the avatar), the effect of that is to suppress the presence dot.

I think we should fix that by passing the email through and showing the presence dot. I don't see any sign that the omission was intentional; and looking around (with the help of `rg -B2 -A5 '<UserAvatar'`) at other places we use either `UserAvatarWithPresence` or plain `UserAvatar`, the choice consistent with the rest of the UI would be to include the presence dot. The only places we don't are (a) in `AccountDetails`, where we show the presence dot elsewhere as part of a more space-occupying UI; and (b) for the user themself, in `OwnAvatar`.

(This doesn't need to be part of this PR -- would be a fine followup.)

Reply:
OK, sounds good! I'll make a note to return to this as a followup.

Reply:
Opened as https://github.com/zulip/zulip-mobile/pull/4341.